Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -457,6 +457,7 @@ public static void RegisterRoutes(IKernel kernel, RouteCollection routes)
             routes.MapHttpProcessesRoute("one-process-thread", "/{processId}/threads/{threadId}", new { controller = "Process", action = "GetThread" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpProcessesRoute("all-modules", "/{id}/modules", new { controller = "Process", action = "GetAllModules" }, new { verb = new HttpMethodConstraint("GET") });
             routes.MapHttpProcessesRoute("one-process-module", "/{id}/modules/{baseAddress}", new { controller = "Process", action = "GetModule" }, new { verb = new HttpMethodConstraint("GET") });
+            routes.MapHttpProcessesRoute("one-process-dump-on-crash", "/{id}/dump/oncrash", new { controller = "Process", action = "TakeMemoryDumpOnCrash" }, new { verb = new HttpMethodConstraint("POST") });
```

Comment:
yes the `{id}` vs `{processId}` part is ... unfortunate. I can have a separate commit that makes them consistent.

I made it that way mostly because we already had a `/dump` route that takes a dump on demand. so I thought I'd add a `/oncrash` to it. Now that I'll be adding all REST operations to it, not sure which route should we go with exactly. Especially that the dump is actually taken on any process termination, not just crash.....